packages/server/graphql/mutations/deleteUser.ts

```diff
@@ -80,10 +80,14 @@ export default {
       reasonRemoved: validReason,
       updatedAt: new Date()
     }
-    setTimeout(() => {
-      db.write('User', userIdToDelete, update)
-      updateUser(update, userIdToDelete)
-    }, 30000)
+    setTimeout(
+      () =>
+        Promise.all([
+          db.write('User', userIdToDelete, update),
+          updateUser(update, userIdToDelete)
+        ]).then(() => dataLoader.get('users').clear(userIdToDelete)),
```

Comment:
curious, why do we need to clear the dataloader here? it doesn't look like we prime the `users` dataloader anywhere above

Reply:
You're right, with the answer to the question above this does not make sense.